arduino/splitflap/platformio.ini

```diff
@@ -82,9 +82,11 @@ src_filter = +<*> -<.git/> -<.svn/> -<example/> -<examples/> -<test/> -<tests/>
 
 lib_deps =
     TFT_eSPI@2.2.23
+    PubSubClient@2.8
     ${extra.lib_deps}
 
 build_flags =
+    -DMQTT=true
```

Comment:
You might not want this depending on whether you think MQTT or the display will be the primary use case for the ESP32 firmware.

Reply:
I think for now it'd be best to have MQTT default to disabled, only because it requires additional setup to configure at first, which may not be needed for people that are not planning to use MQTT.

I'm planning to design a new central controller PCB soon based around a TTGO T-Display (which is my favorite ESP32 breakout board and includes a small built-in IPS LCD), so my plan is to have a default configuration for that PCB design, but other options easily configurable (MQTT, BLE, adafruit.io, etc).

Reply:
Oooh i'll have to pick up one of those TTGO T-Display boards, they look really nice for the price.

Reply:
Made the change to disable MQTT by default